### PR TITLE
SEO-174775 WPF Redirect  links

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Have changed Page with Redirect links.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/DataGrid/Unbound-Column.md [hunk 1]
```diff
@@ -389,15 +389,15 @@ Renderer
...
+{{'[GridUnBoundCellTemplateRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridUnBoundCellTemplateRenderer.html)'| markdownify }}
 </td>
 </tr>
 <tr>
 <td>
 UnBoundTextColumn
 </td>
 <td>
-{{'[GridUnBoundCellTextBoxRenderer](http://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridUnBoundCellTextBoxRenderer.html)'| markdownify }}
+{{'[GridUnBoundCellTextBoxRenderer](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Cells.GridUnBoundCellTextBoxRenderer.htmlde)'| markdownify }}
```
Comment (Aishwarya-Ganesan) [inline wpf/DataGrid/Unbound-Column.md:400]: redirection link is wrong. end of the link de is wrongly added

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: wpf/DataGrid/Unbound-Column.md (hunk 1 above)
Comment (Yvone-Atieno) [reply wpf/DataGrid/Unbound-Column.md]: updated proper link

**Pull request merged**